application/src/main/java/org/togetherjava/tjbot/commands/basic/TestCommand.java

```diff
@@ -0,0 +1,56 @@
+package org.togetherjava.tjbot.commands.basic;
+
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.components.ActionRow;
+import net.dv8tion.jda.api.interactions.components.Modal;
+import net.dv8tion.jda.api.interactions.components.selections.SelectMenu;
+import net.dv8tion.jda.api.interactions.components.text.TextInput;
+import net.dv8tion.jda.api.interactions.components.text.TextInputStyle;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+
+public final class TestCommand extends SlashCommandAdapter {
```

Comment:
Seems you forgot to remove this leftover

Reply:
I thought I deleted it before pushing, I will clean it up and push along with the changes recommended